vllm/engine/metrics.py

```diff
@@ -120,6 +120,15 @@ def __init__(self, labelnames: List[str], vllm_config: VllmConfig):
             name="vllm:tokens_total",
             documentation="Number of prefill plus generation tokens processed.",
             labelnames=labelnames)
+        self.counter_requests_with_evicted_tokens = self._counter_cls(
+            name="vllm:requests_with_evicted_tokens_total",
+            documentation=
+            "Number of requests that had tokens evicted from KV cache",
+            labelnames=labelnames)
+        self.counter_total_evicted_tokens = self._counter_cls(
+            name="vllm:total_evicted_tokens_total",
```

Comment:
- we should call this `total_preempted_tokens` for consistency with `vllm:num_preemptions_total`